### PR TITLE
Service Unlink and Delete

### PR DESCRIPTION
**Pull request by nzoschke**

### CODE REVIEW

File: models/process.go [hunk 1]
```diff
@@ -52,20 +52,22 @@ func ListProcesses(app string) (Processes, error) {
...
 			ps = append(ps, Process{
 				App:   app,
 				Name:  *cd.Name,
 				Count: 1,
 			})
+
+			for _, l := range cd.Links {
+				ls := strings.Split(*l, ":")
+
+				fmt.Printf("%+v\n", ls)
```
Comment (ddollar) [inline models/process.go]: Looks like some debugging made it through

File: models/service.go [hunk 2]
```diff
@@ -77,6 +89,55 @@ func LinkService(app string, process string, stack string) error {
 	return err
 }
 
+func UnlinkService(app string, stack string) error {
```
Comment (ddollar) [inline models/service.go]: This could use `App#UpdateParams` instead

File: models/service.go [hunk 3]
```diff
@@ -77,6 +89,55 @@ func LinkService(app string, process string, stack string) error {
...
+	a, err := GetApp(app)
+
+	if err != nil {
+		return err
+	}
+
+	r, err := a.LatestRelease()
```
Comment (ddollar) [inline models/service.go]: This could use `App#UpdateParams` instead

File: views/app.tmpl [hunk 4]
```diff
@@ -70,7 +70,7 @@
 								</a>
 
 								{{ if .ServiceType }}
-									<button class="btn btn-xs btn-info service-link pull-right" title="Service for {{ .Name }}" data-app="{{ .App }}" data-name="{{ .Name }}" data-type="{{ .ServiceType }}" data-toggle="modal" data-target="#service-link">Link</button>
+									<button class="btn btn-xs btn-primary service-link pull-right" title="Service for {{ .Name }}" data-app="{{ .App }}" data-name="{{ .Name }}" data-type="{{ .ServiceType }}" data-toggle="modal" data-target="#service-link">Link</button>
```
Comment (ddollar) [inline views/app.tmpl]: I think this looked better as `-info`

**Comment by nzoschke** [comment]
screen shot 2015-06-08 at 3 44 05 pm

File: views/app.tmpl (hunk 4 above)
Comment (ddollar) [inline views/app.tmpl]: Nevermind, didn't realize ports were green now. Looks good.

Comment (ddollar) [inline views/app.tmpl]: Maybe the ports should be `-info` actually.

Comment (nzoschke) [inline views/app.tmpl]: Yeah, there's more than a few places where we are using the wrong bootstrap class. Things that perform actions should be primary, things that are artifacts should be info, etc.

**Comment by ddollar** [comment]
:+1:

**Pull request merged**